mysql-test/suite/rocksdb.sys_vars/t/rocksdb_max_open_files_basic.test

```diff
@@ -4,3 +4,35 @@
 --let $read_only=1
 --let $session=0
 --source ../include/rocksdb_sys_var.inc
+
+
+# MYR-179 : Basic Sysbench run fails with basic MyROCKS install due to lack of open files
```

Comment:
OK but in this other PR I just reviewed a similar test went to a separate test in the main suite.

Reply:
Yeah, you're right. I actually did it in the other work as a result of discussing it elsewhere, which was probably here. I'll split this out so we stay somewhat consistent.